rose/server/score_test.py

```diff
@@ -1,8 +1,8 @@
 from rose.common import actions, config, obstacles
-import track
-import player
-import score
-import pytest
+from . import track
+from . import player
+from . import score
+from . import pytest
```

Comment:
But pytest is not a module this package, but a 3rd party module. This change is not needed.

Reply:
First, how can I run the tests on my machine?
I don't have for some reason pytest on the virtualenv (or should I run some other command?)

I fixed the import and did another commit and push but not seeing it here, should I do another pull request?

Reply:
To install the development packages:

    pipenv install --dev

(We need to document this in the readme)

If you fix this in the next commit, pushing again to github should show it here.

If you amended the original commit, you need to push with --force to replace the original commit on gituhub,

Your PR will be updated automatically in both ways.